airbyte-scheduler/client/src/main/java/io/airbyte/scheduler/client/DefaultSynchronousSchedulerClient.java

```diff
@@ -62,11 +64,14 @@ public SynchronousResponse<StandardCheckConnectionOutput> createSourceCheckConne
         .withConnectionConfiguration(sourceConfiguration)
         .withDockerImage(dockerImage);
 
+    final UUID jobId = UUID.randomUUID();
+    final ConnectorJobReportingContext jobReportingContext = new ConnectorJobReportingContext(jobId, dockerImage);
```

Comment:
I think we are running into one of the oddities due to Temporal. This UUID that is mislabeld as `jobIb` in a lot of functions is actually a `workflowId` from a Temporal point of view.
If we report that as `jobIb`, it might end up being confusing because we have a jobs table with `jobId`s that are actual ints. My understanding is that since we do not always wait for a job to actually be started, we do not have a proper jobId yet.
I think we should track and report them as `workflowId`s to reduce the confusion.

Reply:
Hmm, I opened up the temporal UI to check this out and it seems like this UUID is passed on as a jobId and workflowId is something else:

![image](https://user-images.githubusercontent.com/4368928/182200925-66ff4356-ca19-4299-9269-4abb3d73db17.png)

(notice workflowId is a different UUID) - I confirmed the `jobId` is the one being reported to sentry in the current implementation. There do seem to be two "formats" for job Ids though, one is a long used in sync jobs and then there's these UUIDs used in the other synchronous connector jobs (check/spec/discover)

Reply:
@benmoriceau any thoughts here RE: jimmy's comments on this actually being a workflow ID?

Reply:
It is not a workflow Id, it is needed to be able to create the s3 bucket that will contain the job logs (see: https://console.cloud.google.com/storage/browser/prod-ab-app-logs/job-logging/workspace?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&authuser=1&project=prod-ab-cloud-proj&supportedpurview=project&prefix=&forceOnObjectsSortingFiltering=false). It says it is a `workspace` folder but it is not, for example if you search for `442761 ` you will find the logs for one attempt of https://cloud.airbyte.io/workspaces/bd7f276c-fb96-46bf-8f3c-53d194add358/connections/547bd37c-1652-45e0-9095-c1663870012d/status.